css/style.css

```diff
@@ -864,3 +866,18 @@ grid-template-columns: 1fr 1fr;
 .stars{
   text-align: right;
 }
+
+@media (max-width: 430px) {
```

Comment:
Try to adjust to our @media breakpoints.
Here is an example from our code:

breakpoint for tablet starts here
```
@media (min-width: 768px) {
}
```
breakpoint for desktop starts here
```
@media (min-width: 1200px) {
 }
```

Reply:
I just ajusted to the mobile L and smaller because was the only one that was "deforming" the page.